ssh_check/tests/test_integration.py

```diff
@@ -5,23 +5,27 @@
 
 import pytest
 
+from datadog_checks.ssh_check import CheckSSH
+
 from . import common
 
 pytestmark = pytest.mark.integration
 
 
 @pytest.mark.usefixtures("dd_environment")
-def test_check(aggregator, check, instance):
-    check.check(instance)
+def test_check(aggregator, instance):
+    check = CheckSSH('ssh_check', {}, [instance])
+    check.check(None)
```

Comment:
I get that `instance` should not be used by `.check()` but should we stick to what the Agent does?

```suggestion
    check.check(instance)
```